extensions/wikia/CreateNewWiki/CreateNewWikiHooks.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+class CreateNewWikiHooks {
+
+	public static function onMakeGlobalVariablesScript( array &$vars, OutputPage $outputPage ) {
+		if ( $outputPage->getTitle()->isSpecial( 'CreateNewWiki' ) ) {
+			global $wgWikiaBaseDomain, $wgCreateLanguageWikisWithPath;
+
+			$vars['wgWikiaBaseDomain'] = $wgWikiaBaseDomain;
+			$vars['wgCreateLanguageWikisWithPath'] = $wgCreateLanguageWikisWithPath;
+		}
```

Comment:
`OutputPage::addJsConfigVars` allows you to emit JS variables when rendering a special page. No need to register a "generic" hook with a check for a specific special page inside.